bitnami/prometheus-operator/values.yaml

```diff
@@ -548,6 +548,8 @@ prometheus:
       ##
       # pullSecrets:
       #   - myRegistryKeySecretName
+    
+    extraArgs: [] 
```

Comment:
It would be nice if you can add a comment here and an example if possible, it is really helpful for the rest of the users.

Reply:
I added some comments and some example cases.